pom.xml

```diff
@@ -121,6 +121,11 @@
                 <groupId>org.eluder.coveralls</groupId>
                 <artifactId>coveralls-maven-plugin</artifactId>
                 <version>${coveralls.version}</version>
+                <configuration>
+                    <excludes>
+                        <exclude>io.github.ztkmkoo.dss.example/**/*</exclude>
+                    </excludes>
+                </configuration>
```

Comment:
Do you mean that you want to exclude test from jacoco report?
Ref https://github.com/trautonen/coveralls-maven-plugin.
The plugin for submitting Java code coverage reports to Coveralls web service. == only submitt jacoco report to coveralls

Reply:
I just think that there are no @test in example, so jacoco reports is already skipped. So I insert in coveralls part. 

And it's my bad. I apologize. Thanks to your ref, I see that inserting configuration to coveralls is not good. So I'll move this exclude configuration to jacoco now.